style.css

```diff
@@ -0,0 +1,598 @@
+* {
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/207967751-50722f9c-d9d0-4559-b7fe-70e69feec192.png)

Please make sure to add [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) to your project 😄. This will enable you to create two designs: one for mobile and another one for desktop. Keep in mind that these designs still should be on the _same_ project 😉. When the user's device `width` is small enough (768px recommended) it will use the mobile query, while larger devices will be shown the desktop query.

Reply:
- Hey, as a project requirement and as requested by the previous reviewer, kindly add a media query so you can add the CSS rule for your desktop version. Currently, you've hardcoded the part for the desktop version and this had made your website not responsive on mobile. please, kindly adjust your styling by using media query and fix the issue with the responsiveness of your website. See the image of your mobile design below:



![Screenshot from 2022-12-15 23-53-59](https://user-images.githubusercontent.com/59397159/207984174-09b862b5-6888-409b-8e34-72920c9d7e35.png)
![Screenshot from 2022-12-15 23-53-52](https://user-images.githubusercontent.com/59397159/207984193-d7104d83-ae23-4589-9dbc-19d5c00666b7.png)
![Screenshot from 2022-12-15 23-53-37](https://user-images.githubusercontent.com/59397159/207984204-04480239-921e-4be6-84d8-dc924b495042.png)